### PR TITLE
Update GettingStarted.md

### PR DESCRIPTION
**Pull request by raxityo**
Fix example of `URLSession` extension to make it compile.

### CODE REVIEW

File: Documentation/GettingStarted.md [hunk 1]
```diff
@@ -557,28 +557,30 @@ This is how HTTP requests are wrapped in Rx. It's pretty much the same pattern l
 
 ```swift
 extension Reactive where Base: URLSession {
-    public func response(_ request: URLRequest) -> Observable<(Data, HTTPURLResponse)> {
+    public func response(request: URLRequest) -> Observable<(response: HTTPURLResponse, data: Data)> {
         return Observable.create { observer in
-            let task = self.dataTaskWithRequest(request) { (data, response, error) in
+
+            let task = self.base.dataTask(with: request) { (data, response, error) in
+
```
Comment (freak4pc) [inline Documentation/GettingStarted.md]: Can you please remove the two extra empty lines here? (or at least the one above)

File: Documentation/GettingStarted.md [hunk 2]
```diff
@@ -557,28 +557,30 @@ This is how HTTP requests are wrapped in Rx. It's pretty much the same pattern l
...
+            let task = self.base.dataTask(with: request) { (data, response, error) in
+            
                 guard let response = response, let data = data else {
-                    observer.on(.error(error ?? RxCocoaURLError.Unknown))
+                    observer.on(.error(error ?? RxCocoaURLError.unknown))
                     return
                 }
 
+
```
Comment (freak4pc) [inline Documentation/GettingStarted.md]: Same

File: Documentation/GettingStarted.md [hunk 3]
```diff
@@ -557,28 +557,30 @@ This is how HTTP requests are wrapped in Rx. It's pretty much the same pattern l
...
                 guard let httpResponse = response as? HTTPURLResponse else {
                     observer.on(.error(RxCocoaURLError.nonHTTPResponse(response: response)))
                     return
                 }
 
-                observer.on(.next(data, httpResponse))
+
+                observer.on(.next((httpResponse, data)))
```
Comment (freak4pc) [inline Documentation/GettingStarted.md]: Same

File: Documentation/GettingStarted.md [hunk 4]
```diff
@@ -557,28 +557,30 @@ This is how HTTP requests are wrapped in Rx. It's pretty much the same pattern l
...
                 observer.on(.completed)
             }
 
             task.resume()
 
-            return Disposables.create {
-                task.cancel()
-            }
+            return Disposables.create(with: task.cancel)
```
Comment (freak4pc) [inline Documentation/GettingStarted.md]: It's better to leave the previous syntax - it's more commonly used and a matter of preference, not an actual fix.

Comment (freak4pc) [reply Documentation/GettingStarted.md]: This line still needs to be reverted

**Comment by raxityo** [comment]
Thanks, @freak4pc!
Updated based on the suggestions.

**Pull request merged**